src/core/module/image/ImageService.ts

```diff
@@ -360,4 +361,45 @@ export class ImageService {
 
     return ImageResponse.convertFromImage(detail_image).toJson();
   }
+
+  async changeVisibilityImage(user_id: number, image_id: number): Promise<void> {
+    const image = await this.imageRepository.getById(image_id);
+
+    if (!image) {
+      throw new Exception(ImageError.IMAGE_NOT_FOUND);
+    }
+
+    if (image.userId !== user_id) {
+      throw new Exception(ImageError.FORBIDDEN_IMAGES);
+    }
+
+    const new_visibility = !image.visibility;
+
+    try {
+      const updated_image = await this.imageRepository.updateVisibilityById(
```

Comment:
đặt biến updated_image không dùng thì await this.imageRepository.updateVisibilityById( được rồi nhé

Reply:
ok nhé